rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -1629,18 +1726,27 @@ void main_window::HandlePupInstallation(const QString& file_path, const QString&
 		});
 
 		// Wait for the completion
-		for (uint value = progress.load(); value < update_filenames.size(); std::this_thread::sleep_for(5ms), value = progress)
+		uint value{};
```

Comment:
Why is this not a local variable in the lambda?

Reply:
Fixed